defaults/main.yml

```diff
@@ -18,3 +18,12 @@ drbd_interface: enp0s8
 drbd_network_shared_secret: wXE8MqVa
 
 drbd_vip: 192.168.250.100
+
+# Debian9
+drbd_additional_packages: ['drbd8-utils']
```

Comment:
We could change this to `drbd_debian_additional_packages` for Debian based apt packages. This would allow the flexibility to not have to define this variable differently between Debian and RHEL based OS's.